### ISSUE TITLE
Android Autolinking Docs & Breaking Change

### ISSUE THREAD

**Issue by jspizziri** [Nozbe/WatermelonDB#958]
It appears that the recent implementation of Android autolinking has:

1. Left outdated install instructions. (WIP PR with fix https://github.com/Nozbe/WatermelonDB/pull/959)
2. Introduced a BC Breaking change (at least it did in our project). (PR with fix https://github.com/Nozbe/WatermelonDB/pull/960)

The work done on https://github.com/Nozbe/WatermelonDB/pull/805 and https://github.com/Nozbe/WatermelonDB/pull/884 introduced autolinking for Android. However, documentation [which was added to the `CHANGELOG` should've been added to the install documentation](https://github.com/Nozbe/WatermelonDB/pull/805/files#diff-bd3b90ea5e059997a16d0b05acfe64b7eeb85096cb700fc14fe7a76f0f8f1002R13). Additionally, the detail contained in the comment seems to have been removed by [a later commit which updated the `CHANGELOG`](https://github.com/Nozbe/WatermelonDB/pull/791/commits/813bb6b770f4499e5cc287c24daf716412088200#diff-bd3b90ea5e059997a16d0b05acfe64b7eeb85096cb700fc14fe7a76f0f8f1002L16). Lastly, the change appears to be a breaking one for anyone who had manually installed WatermelonDB prior to autolinking upon upgrade (at least in our project).

The Breaking Change error we're seeing in our project is the following:

```
Fatal Exception: java.lang.IllegalStateException
Native module DatabaseBridge tried to override DatabaseBridge. Check the getPackages() method in MainApplication.java, it might be that module is being created twice. If this was your intention, set canOverrideExistingModule=true. This error may also be present if the package is present only once in getPackages() but is also automatically added later during build time by autolinking. Try removing the existing entry and rebuild.
```

I have submitted two separate PR's one to address the install documentation and the other to address the Breaking Change in the `CHANGELOG`.

**Comment by radex** [comment]
Closing in favor of only tracking your pending PR, thank you! ❤️
